common-items.lic

```diff
@@ -119,9 +119,44 @@ module DRCI
     result =~ /inside/
   end
 
+  # Is the item in either hand?
   def in_hands?(item)
-    item_regex = item.split.size > 1 ? /.*\b#{item.split.first}.*\b#{item.split.last}/i : /\b#{item}/i
-    item_regex =~ "#{DRC.right_hand}#{DRC.left_hand}"
+    in_hand?(item, 'either')
+  end
+
+  # Is the item in your left hand?
+  def in_left_hand?(item)
+    in_hand?(item, 'left')
+  end
+
+  # Is the item in your right hand?
+  def in_right_hand?(item)
+    in_hand?(item, 'right')
+  end
+
+  # Checks if the item is in one or more hands.
+  # Hand options are: left, right, either, both.
+  def in_hand?(item, whichHand = 'either')
+    item_regex = item.split.size > 1 ? /#{item.split.first}.*\b#{item.split.last}/i : /\b#{item}/i
+    item_found = false
+    case whichHand.downcase
+    when 'left'
+      # Your holding the item in your left hand?
+      item_found = ("#{DRC.left_hand}" =~ item_regex) != nil
+    when 'right'
+      # Your holding the item in your right hand?
+      item_found = ("#{DRC.right_hand}" =~ item_regex) != nil
+    when 'either'
+      # At least one hand is holding the item?
+      item_found = in_hand?(item, 'left') || in_hand?(item, 'right')
```

Comment:
Oh, worlds most trivial comment I forgot. If you used your helper methods you made and later change the expected value of 'right' you'll have a few less places to change the hard coded string.

`item_found = in_right_hand?(item) || in_left_hand?(item)`